src/cli.py

```diff
@@ -92,16 +92,20 @@ def _wait_maintenance_disabled(zk, timeout):
     logging.info('Success')
 
 
+def enable_maintenance(zk, timeout, wait_all):
+    zk.ensure_path(zk.MAINTENANCE_PATH)
+    zk.noexcept_write(zk.MAINTENANCE_PATH, 'enable', need_lock=False)
+    if wait_all:
```

Comment:
NIT: This check suggests that `wait_all` is more important than `timeout`, so `wait_all` should come first, and timeout should have a default, thus can be omitted.